db/migrate/20221019121936_add_description_and_url_to_learning_units.rb

```diff
@@ -0,0 +1,6 @@
+class AddDescriptionAndUrlToLearningUnits < ActiveRecord::Migration[7.0]
+  def change
+    add_column :learning_units, :description, :string
```

Comment:
As discussed, if description is not meant to be nullable, it should be included in this migration, along with the changes to the seed file.

Reply:
The job is done.